data-model/shared-models/schema/agent.json

```diff
@@ -21,7 +21,7 @@
     "agentName": {
       "type": "string",
       "examples": [
-        "John Smits"
+        "John Smith"
```

Comment:
wow

Reply:
is this a joke I'm missing? :sweat_smile:

Reply:
Might be missing it as well. Maybe it is an inside joke :smile:

Reply:
Just admiring the attention to detail